server/rating_service/rating_service.py

```diff
@@ -141,7 +138,7 @@ async def _get_player_rating(
             )
             raise ServiceNotReadyError("RatingService not yet initialized.")
 
-        rating_type_id = self._rating_type_ids.get(rating_type.value)
+        rating_type_id = self._rating_type_ids.get(rating_type)
```

Comment:
Imho, you're mixing `rating_type` and `rating_type.value`. please review access to that `PlayerRatings` and `RatingTypeMap`. Once it's map with access by enum (i mean this row), once it's access by value: `self.ratings[RatingType.GLOBAL]` for example (https://github.com/FAForever/server/pull/428/files#diff-2cfcdbd3376b08b6aef38c2858475e9fR192).

Reply:
Yes that is the point of using a `RatingTypeMap` instead of just a normal defaultdict.